proto/info/patch_envelope.proto

```diff
@@ -55,28 +45,15 @@ message ZInfoPatchEnvelope {
     repeated string errors = 6;
 }
 
-message ZInfoOpaqueAppInstanceStatus {
-    string appId = 1;
-    string appName = 2;
-    EVE_OPAQUE_STATUS_ACTION opaqueStatusAction = 3;
-    bytes opaqueStatus = 4;
-}
-
 // Provide information on PatchEnvelope usage per app
 message ZInfoPatchEnvelopeUsage {
-    // count the number of times app instance called patch APIs
-    uint64 patchApiCallCount = 1;
-    // information about patchEnvelope referred to this app
-    ZInfoPatchEnvelopeApp patchStatus = 2;
-}
-
-// Statistics on usage of specific patchEnvelope for specific app
-message ZInfoPatchEnvelopeApp {
     // UUID of PatchEnvelope
     string uuid = 1;
     // Version of PatchEnvelope
     string version = 2;
-    // Number of times this PatchEnvelope
-    // was sucessfully downloaded
-    uint64 timesDownloaded = 3;
+    // count the number of times app instance called patch APIs
+    uint64 patchApiCallCount = 3;
+    // count the number of times app instance actually downloaded
+    // whole patch envelope or part of it
+    uint64 timesDownload = 4;
```

Comment:
Both of them have the semantics of "counter", but the naming convention does not tell that at all - "times" vs. "count". Can we make the names indicate that both are counters?

Reply:
Changed to snake_case with count in name